README.md

```diff
@@ -1,26 +1,14 @@
-# app
+# Brewhops App [![CircleCI](https://circleci.com/gh/brewhops/app.svg?style=svg)](https://circleci.com/gh/brewhops/app) [![codecov](https://codecov.io/gh/brewhops/app/branch/master/graph/badge.svg)](https://codecov.io/gh/brewhops/app)
 
-## Installs dependencies
-```
-npm i
-```
+The app is written in VueJs and bootstrapped by the vue cli. If you need to add any additional webpack configuration you can extend the exported object in `vue.config.js`. If contributing, please write unit tests for any feature that you add, we're doing ourselves a favor!
 
-### Compiles and hot-reloads for development
-```
-npm run dev
-```
+## Commands
+- `npm ci`: preferred because it will install dependencies based on package-lock.json, enforcing consistent package versions
+- `npm run dev`: starts the dev server and hot-reloads on file changes
+- `npm run build`: builds and minifies the project for production
+- `npm run test`: runs unit tests and displays code coverage
+- `npm run lint`: runs the linter on the project to enforce consistent code styling
 
-### Compiles and minifies for production 
-```
-npm run build
-```
-
-### Runs unit tests
-```
-npm run test
-```
-
-### Lints and fixes files
-```
-npm run lint
-```
\ No newline at end of file
+## Deployment Commands 
+- `npm run deploy` - __Please don't run this!__ it will deploy to production
```

Comment:
This seems like a great message to have here, but I think it'd be good to have ruining prod more than a bad tab-complete away. Or at least 1 to 2 "are you sure?"s in there

Reply:
It may be a good idea to write a bash script that adds that check in for us. I'll probably do that. Deploying to production is definitely supposed to be a Dan or Brennan thing. Mainly for cost reasons.